main/data/conversation_list_titlebar_csd.ui

```diff
@@ -25,8 +25,8 @@
         </child>
         <child>
             <object class="GtkMenuButton" id="menu_button">
-                <property name="valign">center</property>
                 <property name="visible">True</property>
+                <property name="margin-end">5</property>
```

Comment:
Other applications that have a horizontal line going through the headerbar (Geary, GNOME Settings, GNOME Contacts) don't set any margin on the buttons next to the line. So I guess we should leave that up to the theme, too.